podcasts/End of Year/Stories/YearOverYearStory.swift

```diff
@@ -0,0 +1,211 @@
+import SwiftUI
+import PocketCastsServer
+import PocketCastsDataModel
+
+struct YearOverYearStory: ShareableStory {
+    var duration: TimeInterval = 5.seconds
+
+    let identifier: String = "year_over_year"
+
+    let yearOverYearListeningTime: YearOverYearListeningTime
+
+    let subscriptionTier: SubscriptionTier = SubscriptionHelper.subscriptionTier
+
+    var title: String {
+        let listeningPercentage = yearOverYearListeningTime.percentage
+        switch listeningPercentage {
+        case _ where listeningPercentage == .infinity:
+            return L10n.eoyYearOverYearTitleSkyrocketed
+        case _ where listeningPercentage > 10:
+            return L10n.eoyYearOverYearTitleWentUp("\(listeningPercentage.clean)%")
+        case _ where listeningPercentage < 0:
+            return L10n.eoyYearOverYearTitleWentDown
+        default:
+            return L10n.eoyYearOverYearTitleFlat
+        }
```

Comment:
We can achieve the same effect without the `where` by doing:

```suggestion
        switch yearOverYearListeningTime.percentage {
        case .infinity:
            L10n.eoyYearOverYearTitleSkyrocketed
        case 10...:
            L10n.eoyYearOverYearTitleWentUp("\(yearOverYearListeningTime.percentage.clean)%")
        case ...0:
            L10n.eoyYearOverYearTitleWentDown
        default:
            L10n.eoyYearOverYearTitleFlat
        }
```